patches.json

```diff
@@ -101,5 +101,132 @@
+        }
+      }
+    }
+  },
+  "MDVA-29996": {
+    "magento/magento2-ee-base": {
+      "Fixes the issue where after enabling category permission, the category page is not getting cached by Full Page Cache.": {
+        ">=2.3.3 <=2.4.0": {
+          "file": "commerce/MDVA-29996__fixes_page_cache_identifier_with_enabled_category_permissions_and_fpc__2.3.3.patch"
+        }
+      }
+    }
+  },
+
+  "MDVA-30164": {
+    "magento/magento2-base": {
+      "Fixes the issue where customers records cannot be exported from the Customers grid, if custom customer attributes exist.": {
+        ">=2.3.1 <=2.4.0": {
+          "file": "os/MDVA-30164__fixes_customer_export_when_custom_customer_attribute_exists__2.3.2.patch"
+        }
+      }
+    }
+  },
+  "MDVA-30444": {
+    "magento/magento2-base": {
+      "Fixes the issue where no confirmation email is sent for orders placed using GraphQL.": {
+        ">=2.3.0 <=2.4.0": {
+          "file": "os/MDVA-30444__fixes_confirmation_email_not_sent_with_orders_placed_using_graphql__2.3.5-p1.patch"
+        }
+      }
+    }
+  },
+  "MDVA-30490": {
+    "magento/magento2-base": {
+      "Fixes the issue where products comparison throws the 500 error page, if one of the products has an empty short description.": {
+        ">=2.3.4 <=2.3.5-p2": {
+          "file": "os/MDVA-30490__fixes_500_error_on_product_comparison_when_having_empty_short_product_description__2.3.5-p2.patch"
+        }
+      }
+    }
+  },
+  "MDVA-30232": {
+    "magento/magento2-base": {
+      "Fixes the issue where it is not possible to re-order if the original order contains a gift card.": {
+        ">=2.3.1 <=2.4.0": {
+          "file": "os/MDVA-30232__fixes_issue_with_re-order_when_order_contains_giftcard__2.3.5-p1.patch"
+        }
+      }
+    }
+  },
+  "MDVA-29965": {
+    "magento/magento2-base": {
+      "Fixed the issue when Invalid Form Key error when adding to cart before page fully refreshes": {
+        ">=2.3.3 <=2.3.5-p2": {
+          "file": "os/MDVA-29965_fixes_invalid_form_key_error_when_adding_to_cart_before_page_fully_refreshes_2.3.5-p1.patch"
+        }
+      }
+    }
+  },
+  "MDVA-30008": {
+    "magento/magento2-b2b-base": {
+      "Fixes the B2B issue where it is not possible to place orders through Admin API for a product which is in a public catalog.": {
+        ">=1.1.1 <=1.1.5-p1 || 1.2.0": {
+          "file": "commerce/MDVA-30008_fixes_order_placement_via_admin_api_for_public_catalog_products_2.3.3.patch"
+        }
+      }
+    }
+  },
+  "MDVA-22469": {
+    "magento/magento2-base": {
+      "Fixes the issue where after upgrading to Magento 2.3.3, Page Builder is not working in the Admin panel and some JS and CSS files are not loading.": {
+        ">=2.3.2-p2 <=2.3.3-p1": {
+          "file": "os/MDVA-22469_fixes_page_builder_css_and_js_not_loading_2.3.3.patch"
+        }
+      }
+    }
+  },
+  "MDVA-22469": {
```

Comment:
Duplicate keys
It should be like
` "MDVA-22469": {
    "magento/magento2-base": {
      "Fixes the issue where after upgrading to Magento 2.3.3, Page Builder is not working in the Admin panel and some JS and CSS files are not loading.": {
        ">=2.3.2-p2 <=2.3.3-p1": {
          "file": "os/MDVA-22469_fixes_page_builder_css_and_js_not_loading_2.3.3.patch"
        }
      }
    },
    "magento/module-page-builder": {
      "Fixes the issue where after upgrading to Magento 2.3.3, Page Builder is not working in the Admin panel and some JS and CSS files are not loading.": {
        ">=1.0.3 <=1.1.1": {
          "file": "commerce/MDVA-22469_fixes_page_builder_css_and_js_not_loading_2.3.3.patch"
        }
      }
    }
  }`

Reply:
fixed